source/MRMesh/MRProgressCallback.h

```diff
@@ -9,4 +10,6 @@ namespace MR
 /// \ingroup BasicStructuresGroup
 typedef std::function<bool( float )> ProgressCallback;
 
+const std::string cLoadingCanceledStr = std::string( "Loading canceled" );
```

Comment:
we have some function to return this line already

Reply:
fixed